src/main/java/Book.java

```diff
@@ -18,7 +15,7 @@
     public Book() {
     }
 
-    public boolean getId() {
-        return false;
+    public long getId() {
+        return this.id;
     }
```

Comment:
Not sure why `getId()` was returning a `boolean`
I changed it to return the actual id which is a `long`